source/install/tarball.rst

```diff
@@ -8,8 +8,8 @@ You can find links to the binary tarballs under the *Generic Linux* menu item on
 
 There are two tarballs available:
 
-- ``percona-server-mongodb-4.4.1-2-x86_64.glibc2.17.tar.gz`` is the general tarball, compatible with any operation system but for Centos 6.
-- ``percona-server-mongodb-4.4.1-2-x86_64.glibc2.12.tar.gz`` is the tarball for Centos 6.  
+- ``percona-server-mongodb-4.4.1-2-x86_64.glibc2.17.tar.gz`` is the general tarball, compatible with any `supported operating system <https://www.percona.com/services/policies/percona-software-support-lifecycle#mongodb>`_ but for CentOS 6.
```

Comment:
Change the "but for" to either "but not" or "excluding" or "except"

Reply:
Fixed